### ISSUE TITLE
Doesn't recognize $$props

### ISSUE THREAD

**Issue by tv42** [sveltejs/eslint-plugin-svelte3#148]
It seems the plugin doesn't understand that `$$props` is a globally defined variable in Svelte templates.

```
.../src/lib/custom_icons/IconBase.svelte
  2:10  error  '$$props' is not defined  no-undef
```

The entirety of that svelte file was

```
<img {...$$props} alt="icon" />

<style lang="scss">
        img {
                width: 100%;
                height: auto;
                max-height: 100%;
        }
</style>
```

Due to https://github.com/sveltejs/eslint-plugin-svelte3/issues/134, there's no way to disable that error either.

**Comment by Conduitry** [comment]
I can't reproduce this. I'll need a more complete reproduction. The way this is implemented is that the Svelte compiler tells this ESLint plugin which implicit magic globals exist in the component and that ESLint should not warn about, and this appears to be working for me.

**Comment by tv42** [comment]
> I'll need a more complete reproduction.

Yeah, understandable. I need to extract it from a larger program.

**Comment by wallw-bits** [comment]
I am also getting this for both `$$props` and `$$restProps`. [Reproduction here](https://github.com/wallw-bits/eslint-svelte-issue-repro).
